test/utils/Extensions.sol

```diff
@@ -214,12 +213,11 @@ contract LendingPoolExtension is LendingPool {
     function getYearlySeconds() public pure returns (uint256) {
         return YEARLY_SECONDS;
     }
-}
 
-contract LendingPoolExtension_NEW is LendingPool_NEW {
-    constructor(ERC20 _asset, address _treasury, address _vaultFactory, address _liquidator)
-        LendingPool_NEW(_asset, _treasury, _vaultFactory, _liquidator)
-    { }
+    function setOpenPosition(address account, uint128 amount) public {
+        //        openPosition[account] = amount;
```

Comment:
Why is this commented out?

Reply:
openPosition variable does not exist, it was more for guideline for me ill delete the comment